packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/CameraSelector.tsx

```diff
@@ -0,0 +1,39 @@
+import { Flex, Label, SelectField, View } from '@aws-amplify/ui-react';
+import React from 'react';
+import { LivenessClassNames } from '../types/classNames';
+
+interface CameraSelectorProps {
+  selectedDeviceId?: string;
+  onCameraChange: (e: React.ChangeEvent<HTMLSelectElement>) => void;
+  selectableDevices: MediaDeviceInfo[];
```

Comment:
IMO we can simplify the naming of the `props` a bit here since camera selection is implied by the component name, e.g.
```ts
  deviceId?: string;
  onSelect?: (e: React.ChangeEvent<HTMLSelectElement>) => void;
  devices: MediaDeviceInfo[];
```

Reply:
on board with this